C/CUDA/CUDA_Driver/init.jl

```diff
@@ -72,8 +72,10 @@ function try_driver(driver, deps)
 
         exit(0)
     """
-    success(`$(Base.julia_cmd()) --compile=min -t1 --startup-file=no -e $script $driver $deps`)
+    # make sure we don't include any system image flags here since this will cause an infinite loop of __init__()
+    success(`$(Cmd(filter(e -> !startswith(r"-J|--sysimage"), Base.julia_cmd().exec))) --compile=min -t1 --startup-file=no -e $script $driver $deps`)
```

Comment:
```suggestion
    success(`$(Cmd(filter(!startswith(r"-J|--sysimage"), Base.julia_cmd().exec))) --compile=min -t1 --startup-file=no -e $script $driver $deps`)
```

Reply:
This is wrong, either you do `e->startswith(e, "")`, or you use the curried version `startswith("")`.